packages/tools/src/tools/annotation/DragProbeTool.ts

```diff
@@ -39,34 +44,138 @@ export default class DragProbeTool extends ProbeTool {
     super(toolProps, defaultToolProps);
   }
 
-  _mouseUpCallback = (
-    evt: EventTypes.MouseUpEventType | EventTypes.MouseClickEventType
-  ) => {
+  addNewAnnotation = (
```

Comment:
This is almost perfect, my only qualm with this now is that since we don't add annotations with this tool, we should swap this callback to `postMouseDownCallback`? Using `addNewAnnotation` implies something will be added to state.